spec.md

```diff
@@ -818,12 +818,10 @@ $ curl http://username:password@service-broker-url/v2/service_instances/:instanc
 | 409 Conflict | MUST be returned if a Service Instance with the same id already exists but with different attributes. The expected response body is `{}`, though the description field MAY be used to return a user-facing error message, as described in [Service Broker Errors](#service-broker-errors). |
 | 422 Unprocessable Entity | MUST be returned if the Service Broker only supports asynchronous provisioning for the requested plan and the request did not include `?accepts_incomplete=true`. The expected response body is: `{ "error": "AsyncRequired", "description": "This Service Plan requires client support for asynchronous service operations." }`, as described below (see [Service Broker Errors](#service-broker-errors). |
 
-Responses with any other status code will be interpreted as a failure and a
-deprovision request MUST be sent to the Service Broker to prevent an orphan
-being created on the Service Broker.
-
-Service Brokers can include a user-facing message in the `description` field;
-for details see [Service Broker Errors](#service-broker-errors).
+Request timeouts, responses with any other status code, or malformed response
+bodies will be interpreted as failures. Service Brokers can include a user-facing
```

Comment:
Anyone else bothered by the use of the word "will"?  I know its not new for this PR, but what do people think about making it a MUST to make it clear that there is no ambiguity about it, it IS A FAILURE. The MUST wouldn't place any new requirements on the Platform w.r.t. what it does about it (it could choose to ignore the failure), but at least from a spec perspective we're very clear on what *we* think of these types of responses. This may not be the only spot in the spec for this either - so if people agree I will open a new PR/issue for it.

Reply:
That seems reasonable, let's open an issue for it!